src/app/connect/connect.component.ts

```diff
@@ -201,13 +201,9 @@ export class ConnectComponent implements OnDestroy {
 
     private save() {
         this._service.save(this._conn);
-
-        this._service.active.subscribe(c => {
-            if (!c) {
-                this._service.connect(this._conn);
-            }
-        }).unsubscribe();
-
+        var sub = this._service.connect(this._conn).subscribe(_ => {}, _ => {}, () =>{
```

Comment:
previously, it seems to reconnect when connection is not established. But you did not implement that. Is this by design?

Reply:
The previous code is just a roundabout way of doing what I wrote. since the subscriber would always see this._conn

However, I am reverting this line since it is out of scope of this PR